src/electionguard_cli/setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
```

Comment:
This shouldn't be necessary. This is the other tool outside poetry. 
We should be wrapping this tool into our existing python package. 
Reach out to me about this.

Reply:
As discussed publishing the CLI so that end users can consume it via PIP Install is a separate issue that I created here: https://github.com/microsoft/electionguard-python/issues/568